docs/basic-features/script.md

```diff
@@ -243,9 +243,8 @@ Or by using the `dangerouslySetInnerHTML` property:
 />
 ```
 
-There are two limitations to be aware of when using the Script component for inline scripts:
+**Note when using the Script component with inline scripts**:
 
-- Only the `afterInteractive` and `lazyOnload` strategies can be used. The `beforeInteractive` loading strategy injects the contents of an external script into the initial HTML response. Inline scripts already do this, which is why **the `beforeInteractive` strategy cannot be used with inline scripts.**
 - An `id` attribute must be defined in order for Next.js to track and optimize the script
```

Comment:
I think you can merge both lines now into a single sentence since there is only a single bullet point:

```
The `id` property is required for **inline scripts** in order for Next.js to track and optimize the script.
```

Reply:
Good call, updated!